deployer/templates/es.yaml

```diff
@@ -41,6 +41,7 @@ objects:
             component: ${ES_CLUSTER_NAME}
             deployment: ${ES_DEPLOYMENT_NAME}
         spec:
+          terminationGracePeriodSeconds: 1800
```

Comment:
We could make this tunable with parameters for the deployer; it wouldn't work for a configmap since this is in the pod spec itself. 
We are just looking to give ES and Fluentd more time to complete their shutdown operations so this is to increase that window of time -- so I'd be cautious of letting users just change this on the fly since that could end up putting ES/Fluentd in a bad state (if they set this to `0` then there is no grace period -- it will SIGKILL right away instead of SIGTERM)